assets/compat.ini

```diff
@@ -0,0 +1,5 @@
+# Fight Night Round 3

```

Comment:
If we do this, I think we need more information on each one:

1. Issue #
2. Basic description of features in the game affected (for repro)
3. Basic technical description of what this change is doing / why it doesn't work without it.

IMHO, that would even go for optimizations, although maybe 3 would be grouped.

-[Unknown]